### PR TITLE
Translate runtime requests

### PR DESCRIPTION
**Pull request by s1lver**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | no

### CODE REVIEW

File: docs/guide-ru/runtime-requests.md [hunk 1]
```diff
@@ -124,3 +124,57 @@ if ($headers->has('User-Agent')) { /* в запросе есть заголов
 $userHost = Yii::$app->request->userHost;
 $userIP = Yii::$app->request->userIP;
 ```
+
+## Доверенные прокси и заголовки <span id="trusted-proxies"></span>
+
+В предыдущем разделе вы видели, как получить информацию о пользователе, такую как хост и IP-адрес.
+Это будет работать из коробки в обычной установке, где один веб-сервер используется для обслуживания веб-сайта.
+Однако если ваше Yii-приложение  работает за обратным прокси-сервером, вам нужно добавить дополнительную конфигурацию для получения этой информации, поскольку прямой клиент теперь является прокси-сервером, а IP-адрес пользователя передается Yii-приложению с помощью заголовка, установленного прокси-сервером.
```
Comment (samdark) [inline docs/guide-ru/runtime-requests.md]:
```suggestion
Однако если ваше приложение работает за обратным прокси-сервером, вам нужно дополнить конфигурацию, поскольку клиентом теперь является прокси-сервер, а IP-адрес пользователя передаётся приложению с помощью заголовка, установленного им.
```

File: docs/guide-ru/runtime-requests.md [hunk 2]
```diff
@@ -124,3 +124,57 @@ if ($headers->has('User-Agent')) { /* в запросе есть заголов
...
+
+Вы не должны слепо доверять заголовкам, предоставленным прокси, если вы явно не доверяете прокси.
+Начиная с 2.0.13, Yii поддерживает настройку доверенных прокси через
```
Comment (samdark) [inline docs/guide-ru/runtime-requests.md]:
```suggestion
Начиная с 2.0.13, Yii поддерживает настройку доверенных прокси через следующие свойства компонента `request`:
```

File: docs/guide-ru/runtime-requests.md [hunk 3]
```diff
@@ -124,3 +124,57 @@ if ($headers->has('User-Agent')) { /* в запросе есть заголов
...
+[[yii\web\Request::trustedHosts|trustedHosts]],
+[[yii\web\Request::secureHeaders|secureHeaders]], 
+[[yii\web\Request::ipHeaders|ipHeaders]] и
+[[yii\web\Request::secureProtocolHeaders|secureProtocolHeaders]]
+свойства компонента `request`.
```
Comment (samdark) [inline docs/guide-ru/runtime-requests.md]:
```suggestion

```

File: docs/guide-ru/runtime-requests.md [hunk 4]
```diff
@@ -124,3 +124,57 @@ if ($headers->has('User-Agent')) { /* в запросе есть заголов
...
+
+Ниже приведена конфигурация компонента `request` для приложения, которое работает за массивом обратных прокси, которые расположены в IP-сети `10.0.2.0/24`:
```
Comment (samdark) [inline docs/guide-ru/runtime-requests.md]:
```suggestion
Ниже приведена конфигурация компонента `request` для приложения, которое работает за рядом обратных прокси, которые расположены в IP-сети `10.0.2.0/24`:
```

File: docs/guide-ru/runtime-requests.md [hunk 5]
```diff
@@ -124,3 +124,57 @@ if ($headers->has('User-Agent')) { /* в запросе есть заголов
...
+```php
+'request' => [
+    // ...
+    'trustedHosts' => [
+        '10.0.2.0/24',
+    ],
+],
+```
+IP-адрес, по умолчанию, отправляется прокси-сервером в заголовке `X-Forwarded-For`, а протокол (`http` или `https`) отправляется в `X-Forwarded-Proto`.
+Если ваши прокси используют разные заголовки, вы можете использовать конфигурацию компонента `request` для их настройки, например:
```
Comment (samdark) [inline docs/guide-ru/runtime-requests.md]:
```suggestion
Если ваши прокси используют другие заголовки, вы можете использовать конфигурацию компонента `request` для их настройки, например:
```

**Review by samdark — CHANGES_REQUESTED** [review]

File: docs/guide-ru/runtime-requests.md (hunk 5 above)
Comment (s1lver) [reply docs/guide-ru/runtime-requests.md]: @samdark   внес исправления по замечаниям

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**